modules/persistent-storage-byok.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// storage/container_storage_interface/persistent-storage-csi-azure.adoc
+// storage/container_storage_interface/persistent-storage-csi-ebs.adoc
+// storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+:_content-type: CONCEPT
+[id="byok_{context}"]
+= User-managed encryption
+
+The user-managed encryption feature allows you to provide keys during installation that encrypt {product-title} node root volumes, and enables the default storage class to use these keys to encrypt provisioned storage volumes. You specify the custom key in the `platform.<cloud_type>.defaultMachinePlatform` field in the install-config YAML file.
```

Comment:
Suggest to change `You specify the custom key` to:
`You must specify the custom key`, as this is the only place that enables the default storage class to be encrypted. Specify the key in other field won't work for this.

Reply:
+1 EBS operator creates two storage class, replaced default storage class by "managed storage class" or "storage class managed by the storage operator".